chore: add more auto label options

ok-coder1 opened the pull request:
I think the other ones should have just done it, but to be sure and to have no risk.

ok-coder1 committed:
chore: add more auto label options to dependencies

i think the other ones should have just done it, but to be sure and no risk

Signed-off-by: okcoder1 <codershub.code@gmail.com>

ok-coder1 committed:
Update labeler.yml

Signed-off-by: okcoder1 <codershub.code@gmail.com>

BrycensRanch commented on .github/labeler.yml:
```diff
@@ -24,17 +24,22 @@ packaging:
 
 dependencies:
-  - head-branch: ['^dep', '^depend', 'depend', 'dependency', 'dependencies']
+  - head-branch: ['^dep', 'dep', '^deps', 'deps', '^depend', 'depend', 'dependency', 'dependencies']
```
head-branch: ['^(deps?|depend(?:ency|encies)?)(/|$)']

Will match all of these and is easier to maintain.

ok-coder1 replied:
Resolved in https://github.com/SnapXL/SnapX/pull/287/commits/8eff5236c7cb9db297e0bb9ef03850a576c869e7.

ok-coder1 committed:
fix: add better regex expressions

Thanks to @BrycensRanch! (I'm pretty sure the `Co-authored-by` field doesn't work as I have signed off and it adds a footer)

Co-authored-by: Brycen G <brycengranville@outlook.com>

Signed-off-by: okcoder1 <codershub.code@gmail.com>
